js/main.js

```diff
@@ -10,18 +10,62 @@ xhr.addEventListener('load', function renderChampions() {
     $columnHalf.className = 'column-half';
 
     var $card = document.createElement('div');
-    $card.className = 'card ' + key;
+    $card.className = 'card';
     $columnHalf.appendChild($card);
 
     var $name = document.createElement('h2');
     $name.textContent = key;
 
+    var $title = document.createElement('h3');
+    $title.textContent = capitalizeFirstLetter(allChampionData.data[key].title);
+
     var $img = document.createElement('img');
     $img.setAttribute('src', 'https://ddragon.leagueoflegends.com/cdn/img/champion/loading/' + key + '_0.jpg');
 
+    $card.appendChild($title);
     $card.appendChild($name);
     $card.appendChild($img);
     $parentRow.appendChild($columnHalf);
   }
 });
 xhr.send();
+
+document.addEventListener('click', handleCardClick);
+function handleCardClick(event) {
+  if (event.target.tagName === 'IMG') {
+    var xhr2 = new XMLHttpRequest();
+    var $parentCard = event.target.closest('div');
+    var $h2 = event.target.previousSibling;
+    var $img = $parentCard.querySelector('img');
+    if (!$parentCard.className.includes('lore')) {
+      xhr2.open('GET', 'http://ddragon.leagueoflegends.com/cdn/11.22.1/data/en_US/champion/' + $h2.textContent + '.json');
+      xhr2.responseType = 'json';
+      xhr2.addEventListener('load', function getChapionData() {
+        var individualChampionData = xhr2.response;
+        var $lore = document.createElement('p');
+        $lore.textContent = individualChampionData.data[$h2.textContent].lore;
+        $parentCard.appendChild($lore);
+        $parentCard.className += ' lore';
+        $img.className = 'dark';
+      });
+      xhr2.send();
+    } else if ($parentCard.className.includes('lore')) {
+      removeLore();
+    }
+  } else if (event.target.tagName === 'P') {
+    removeLore();
+  }
+}
+
+function removeLore() {
+  var $parentCard = event.target.closest('div');
+  var $img = $parentCard.querySelector('img');
+  var $lore = $parentCard.querySelector('p');
+  $parentCard.className = 'card';
+  $lore.remove();
```

Comment:
Are we still removing the element? I thought we had discussed just hiding and showing it.